sound/soc/sof/intel/hda.c

```diff
@@ -428,11 +429,15 @@ static void hda_dsp_dump_ext_rom_status(struct snd_sof_dev *sdev)
 		len += snprintf(msg + len, sizeof(msg) - len, " 0x%x", value);
 	}
 
-	dev_err(sdev->dev, "error: extended rom status:%s", msg);
+	if (hda->boot_iteration == HDA_FW_BOOT_ATTEMPTS)
+		dev_err(sdev->dev, "error: extended rom status:%s", msg);
+	else
+		dev_dbg(sdev->dev, "extended rom status:%s", msg);
```

Comment:
This starts to be really ugly to replicate all the trace prints twice...

Reply:
@kv2019i well, if we get really too many of these, we certainly could define a macro
```
#define sof_dbg_or_err(dev, is_err, fmt, ...) do { if (is_err) dev_err(dev, "error: " fmt, __VA_ARGS__); else dev_dbg(dev, fmt, __VA_ARGS__); } while (0)

sof_dbg_or_err(sdev->dev, hda->boot_iteration == HDA_FW_BOOT_ATTEMPTS, "extended rom status: %s", msg);
```